### DISCUSSION TITLE
Keyboard is not typing in terminal

### DISCUSSION THREAD

**Discussion by randycolitz** [termux/termux-app#3415] [Q&A] [answered]
I get the app from f-droid and it installs and when you run it you get a terminal screen but it doesn't echo back anything you typed, does anyone know of any working apps or disclose the location please?

**Comment by agnostic-apollo** [comment, marked as answer]
https://wiki.termux.com/wiki/Terminal_Settings#Workaround_issues_with_some_keyboards

**Reply by randycolitz** [reply]
I've got it working as it should.
